translations/de.json

```
@@ -0,0 +1,82 @@
+{
+  "_header": "*******************",
+  "_section": "Display Categories",
+  "_header": "*******************",
+    "solar": "Solar",
+    "battery": "Powerwall",
+    "grid": "Stromnetz",
+    "house": "Haus",
+    "car": "Auto lädt",
+
+  "_header": "*******************",
+  "_section": "General UI terms",
+  "_header": "*******************",
+    "yesterday": "Gestern",
+    "today": "Heute",
+    "unknown": "unbekannte Darstellung!",
+
+  "_header": "*******************",
+  "_section": "Car status",
+  "_header": "*******************",
+    "charging_at": "{NAME} lädt auf mit",
+    "charging_at_plural": "{NAME} und {NUM} laden auf mit",
+    "consuming": "{NAME} is consuming",
```

```suggestion
    "consuming": "{NAME} verbraucht",
```
...if I'm extrapolating from House correctly?